Decred Wallet/Features/App Launch/StartScreenViewController.swift

```diff
@@ -30,37 +45,119 @@ class StartScreenViewController: UIViewController {
+        self.animation.repeatCount = 3
+        // Other stuff
+        self.animation.isRemovedOnCompletion = false
+        self.animation.fillMode = CAMediaTimingFillMode.forwards
+        // Set the delegate
+        self.animation.delegate = self
+        self.logo.contentMode = .scaleAspectFit
+        self.logo.layer.add(animation, forKey: "animation")
+    }
+    
+     func animationDidStop(_ anim: CAAnimation, finished animated: Bool) {
+        self.isAnimated = animated
+    }
+    
+    @objc func appMovedToForeground() {
+        if !self.isAnimated {
+            DispatchQueue.main.async {
+                self.revertAnimationPosition()
+                self.view.layoutIfNeeded()
+            }
+        }
+    }
+    
     @IBAction func animatedLogoTap(_ sender: Any) {
+        if self.isAnimated {
+            return
+        }
+        
         self.timer?.invalidate()
         self.startTimerWhenViewAppears = true
-
+        self.logo.layer.removeAllAnimations()
+        
         let settingsVC = SettingsController.instantiate(from: .Settings).wrapInNavigationcontroller()
         settingsVC.modalPresentationStyle = .fullScreen
         self.present(settingsVC, animated: true, completion: nil)
     }
-
+    
+    func setStartupAnimationPosition() {
+        self.imageViewContainer.center.y += initialAnimationToggleValue
+        self.testnetLabel.center.y += initialAnimationToggleValue
+        self.loadingLabel.center.y += initialAnimationToggleValue
+    }
+    
+    func revertAnimationPosition() {
+        self.imageViewContainer.center.y -= self.initialAnimationToggleValue
+        self.testnetLabel.center.y -= self.initialAnimationToggleValue
+        self.loadingLabel.center.y -= self.initialAnimationToggleValue
+    }
+    
+    @IBAction func createNewwallet(_ sender: Any) {
+        Security.spending(initialSecurityType: .password)
+            .requestNewCode(sender: self, isChangeAttempt: false) { pinOrPassword, type, completion in
+                
+                WalletLoader.shared.createWallet(spendingPinOrPassword: pinOrPassword, securityType: type) {
+                    createWalletError in
+                    
+                    if createWalletError != nil {
+                        completion?.displayError(errorMessage: createWalletError!.localizedDescription)
+                    } else {
+                        completion?.dismissDialog()
+                        NavigationMenuTabBarController.setupMenuAndLaunchApp(isNewWallet: true)
+                    }
+                }
+        }
+    }
+    
+    @IBAction func restoreExistingWallet(_ sender: Any) {
+        print("restore")
+        let restoreWalletVC = RestoreExistingWalletViewController.instantiate(from: .WalletSetup)
+        self.navigationController?.pushViewController(restoreWalletVC, animated: true)
+    }
+    
     func loadMainScreen() {
         if !WalletLoader.shared.isInitialized {
+            print("can't init")
```

Comment:
Print `there was an error initializing multiwallet` and get rid of the comment below.